firefox_puppeteer/ui/windows.py

```diff
@@ -42,11 +42,16 @@ def focused_chrome_window_handle(self):
         """
         with self.marionette.using_context('chrome'):
             return self.marionette.execute_script("""
-              Cu.import("resource://gre/modules/Services.jsm");
-              var win = Services.focus.activeWindow;
-              return win.QueryInterface(Ci.nsIInterfaceRequestor)
-                        .getInterface(Ci.nsIDOMWindowUtils)
-                        .outerWindowID.toString();
+              Components.utils.import("resource://gre/modules/Services.jsm");
+
+              let win = Services.focus.activeWindow;
+              if (win) {
+                return win.QueryInterface(Components.interfaces.nsIInterfaceRequestor)
+                          .getInterface(Components.interfaces.nsIDOMWindowUtils)
+                          .outerWindowID.toString();
+              }
+
+              return null;
```

Comment:
Good not to fail completely here, but in the two places this member is used returning None would still result in a test failure, so I guess we need to update those callers to use Wait as well if we're sure this is the race we think it is.